### DISCUSSION TITLE
My Kurviger App expires today

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#396] [Show and tell]
<img height="300" src="https://github.com/devemux86/cruiser/assets/121806238/dd0f3633-a8b7-4e9c-808c-60ce3e26e834"/> . . . <img height="300" src="https://github.com/devemux86/cruiser/assets/121806238/3f465ce7-a222-439a-8823-9ce2b38f9413"/>

In the last 5 years we have successfully ridden a lot of interesting routes together (okay, last year K2 was only there as a backup, but never necessary). **For this I would like to thank Emux** - who else?

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for your support all these years!

Cruiser development with an emphasis on _user personalization_ continues at full speed.
You can all look forward to many new features in the near future. :slightly_smiling_face:
